apps/web/src/components/Player/Player.tsx

```diff
@@ -30,6 +37,7 @@ function Player() {
   const classes = useStyles();
   const videoElRef = useRef<HTMLVideoElement>(null!);
   const { player, disconnect } = usePlayerContext();
+  const { isChatWindowOpen } = useChatContext();
```

Comment:
I don't think the chat stuff needs to be here without a ChatWindow for the viewers. We should probably add it for the viewers soon, but that can be another PR.